app/src/pages/inside/testCaseLibraryPage/emptyState/details/detailsEmptyState.tsx

```diff
@@ -18,25 +18,35 @@ import Parser from 'html-react-parser';
 import { useIntl } from 'react-intl';
 import { EmptyStatePage } from 'pages/inside/common/emptyStatePage';
 import { referenceDictionary } from 'common/utils';
+import { useUserPermissions } from 'hooks/useUserPermissions';
 
 import { messages } from '../messages';
 
 export const DetailsEmptyState = () => {
   const { formatMessage } = useIntl();
+  const { canEditTestCaseScenario } = useUserPermissions();
+
+  const actionButtons = () => {
+    const buttons: Record<string, string | boolean>[] = [];
+
+    if (canEditTestCaseScenario) {
+      buttons.push({
+        isCompact: true,
+        name: formatMessage(messages.editScenario),
+        variant: 'primary',
+      });
+    }
+
+    return buttons;
+  };
```

Comment:
```suggestion
  const getActionButtons = () => canEditTestCaseScenario ? [{
        isCompact: true,
        name: formatMessage(messages.editScenario),
        variant: 'primary',
      }] : [];
```
you can also wrap it with `useCallback`

Reply:
@ViktorSoroka07 I think here is no need in useCallback as `canEditTestCaseScenario` always the same and won't cause unnecessary re-renders